sass/_theme_rst.sass

```diff
@@ -106,19 +106,16 @@
     @extend h2
 
   // This is the #href that shows up on hover. Sphinx's is terrible so I hack it away.
-  h1, h2, h3, h4, h5, h6, dl dt, p.caption
+  h1, h2, h3, h4, h5, h6, dl dt, p.caption, table > caption
     .headerlink
-      display: none
       visibility: hidden
       font-size: 14px
       @extend .fa
       &:after
-        visibility: visible
         content: "\f0c1"
         font-family: FontAwesome
-        display: inline-block
-    &:hover .headerlink
-      display: inline-block
+    &:hover .headerlink:after
+      visibility: visible
 
```

Comment:
LGTM, although I think slightly reducing the link for table captions looks nicer, by adding:
``` sass
  table > caption .headerlink:after
    font-size: 12px
```
Current PR: 
![14px](https://user-images.githubusercontent.com/3284111/33945440-5e7b64ec-e01f-11e7-85a3-761fd00ec693.png)

With `font-size: 12px`:
![12px](https://user-images.githubusercontent.com/3284111/33945475-717157dc-e01f-11e7-8038-8bd9903c9555.png)

Reply:
Added!